LICENSE.md

```diff
@@ -2,6 +2,8 @@ The MIT License (MIT)
 
 Copyright (c) 2023 Dominik 'Disconnect3d' Czarnota
 
+Copyright (c) 2021 xf1les
```

Comment:
I wonder if this is the right way to do this, doesn't this give them copyright over the entire project? Is there a way we're supposed to specify exactly what the copyright is over?

Reply:
Ya wasn't entirely clear to me, I just went off https://law.stackexchange.com/questions/16847/how-should-multiple-copyright-holders-be-credited-using-the-mit-license which says if you derive from someones code and aren't working with them directly you use the way I did it.

Possibly we could do something like:

``` 
The musl heap analysis functionality is derived from muslheap:

Copyright (c) 2021 xf1les

...
```

which is what it suggests in https://opensource.stackexchange.com/questions/10506/how-to-sub-license-a-modified-mit-licensed-project-as-a-copyright.

Reply:
Yeah I'm not sure there's specific guidance on this wrt the MIT license, I've seen a few different ways of doing it.

e.g. Microsoft just concatenate every license into one file: https://github.com/Microsoft/vscode/blob/main/ThirdPartyNotices.txt

I've seen advice stating just add the names (and sometimes dates) to the copyright holder section.
Sometimes (as in the Microsoft example) there is more detail as to what exactly was copied.

For a deeper dive into the MIT license:
https://writing.kemitchell.com/2016/09/21/MIT-License-Line-by-Line.html